src/shortcuts/shortcuts-hoc.tsx

```diff
@@ -35,6 +35,8 @@ export default function shortcutsHOC<P>(ComposedComponent: ComponentType<P> | st
           options={rgShortcutsOptions}
           disabled={rgShortcutsOptions.disabled}
         >
+          {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+          {/* @ts-ignore */}
```

Comment:
What was the TS error here?

Reply:
It's a new error in TS 4.7 https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#stricter-spread-checks-in-jsx

Reply:
Just pin the TS version to `~4.6`, I'll update it later